### ISSUE TITLE
ix-select should not submit form

### ISSUE THREAD

**Issue by danielleroux** [siemens/ix#442]
### What happened?

If you use an `ix-select` inside a form tag and the user select an item from the list. The form is getting submitted, this should not be happen.

### What type of frontend frameware are you seeing the problem on?

Angular, React, JavaScript

### Which version of iX do you use?

v1.4.1

### Code to produce this issue.

```shell
<form onSubmit={() => console.log('Form submitted')}>
        <IxSelect i18nPlaceholder="My placeholder text">
          <IxSelectItem value="1" label="2"></IxSelectItem>
        </IxSelect>
      </form>
```

**Comment by nazlimine** [comment]
Hi, is there a planned date for 1.5.0 milestone?

**Comment by danielleroux** [comment]
@nazlimine will be released with 1.4.2 later this day
